cms/templates/js/course-video-settings-update-org-credentials-footer.underscore

```diff
@@ -0,0 +1,8 @@
+<button class='button-link action-cancel-course-video-settings' aria-describedby='cancel-button-text'>
+    <%- gettext('Reset') %>
+    <span id='updcancel-button-text' class='sr'><%-gettext('Click Reset to discard your changes.') %></span>
```

Comment:
If this is help text, can it just say "Restore default settings" rather than telling the user to click a button?

Reply:
This is screen reader text